packages/components/src/ecosystem_bar/index.tsx

```diff
@@ -1,23 +1,39 @@
 import clsx from 'clsx';
-import PropTypes from 'prop-types';
 import { useRef } from 'react';
-import { useOnceSwipeTracking, useScrolled } from '@/src/ecosystem_bar/hooks';
+import { useOnceSwipeTracking, useScrolled } from './hooks';
 import styles from './index.module.scss';
 
-const EcosystemBar = ({ items, onFirstSwipe, onItemClick }) => {
-  const navRef = useRef(null);
+export type EcosystemBarItem = {
+  key: string,
+  link: string,
+  isActive: boolean,
+  event: string,
+  text: {
+    mobile: string,
+    others: string,
+  },
+};
+
+export type EcosystemBarProps = {
```

Comment:
I would rather start out with not exporting `EcosystembarProps` and `EcosystemBarItem`. We don't really know if we need them separately and if we start exporting them now, then we need a major release to remove them (breaking change).

Reply:
will do